tests/receivers/oracledb/oracledb_test.go

```diff
@@ -29,15 +29,15 @@ import (
 // account for startup time is to wait for the container to be healthy before continuing test.
 var oracledb = []testutils.Container{testutils.NewContainer().WithContext(
 	path.Join(".", "testdata", "server"),
-).WithName("oracledb").WithExposedPorts("1521:1521").WillWaitForHealth(15 * time.Minute)}
+).WithName("oracledb").WithExposedPorts("1521:1521").WillWaitForHealth(45 * time.Minute)}
```

Comment:
I think pulling the image takes the most time here and 45 min is much too long for an integration test. iirc this can be reduced to <5min and if it's flakey we can address in the image directly.